tests/mongodb.nix

```diff
@@ -72,6 +73,7 @@ in {
         machine.succeed("killall -11 mongod")
         import time
```

Comment:
I think that the sleep is not needed anymore when there's a wait_for_unit later

Reply:
I tested without sleep, but unfortunetly it skips the machine.wait_for_unit line otherwise.